libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -77,125 +77,125 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     // @Units: m
     // @Increment: 0.1
     // @User: Advanced
-    AP_GROUPINFO("ALT_OFFSET", 5, AP_Baro, _alt_offset, 0),
+    AP_GROUPINFO("_ALT_OFFSET", 5, AP_Baro, _alt_offset, 0),
 
-    // @Param: PRIMARY
+    // @Param: _PRIMARY
     // @DisplayName: Primary barometer
     // @Description: This selects which barometer will be the primary if multiple barometers are found
     // @Values: 0:FirstBaro,1:2ndBaro,2:3rdBaro
     // @User: Advanced
-    AP_GROUPINFO("PRIMARY", 6, AP_Baro, _primary_baro, 0),
+    AP_GROUPINFO("_PRIMARY", 6, AP_Baro, _primary_baro, 0),
 
-    // @Param: EXT_BUS
+    // @Param: _EXT_BUS
     // @DisplayName: External baro bus
     // @Description: This selects the bus number for looking for an I2C barometer. When set to -1 it will probe all external i2c buses based on the GND_PROBE_EXT parameter.
     // @Values: -1:Disabled,0:Bus0,1:Bus1
     // @User: Advanced
-    AP_GROUPINFO("EXT_BUS", 7, AP_Baro, _ext_bus, -1),
+    AP_GROUPINFO("_EXT_BUS", 7, AP_Baro, _ext_bus, -1),
 
-    // @Param: SPEC_GRAV
+    // @Param: _SPEC_GRAV
     // @DisplayName: Specific Gravity (For water depth measurement)
     // @Description: This sets the specific gravity of the fluid when flying an underwater ROV.
     // @Values: 1.0:Freshwater,1.024:Saltwater
-    AP_GROUPINFO_FRAME("SPEC_GRAV", 8, AP_Baro, _specific_gravity, 1.0, AP_PARAM_FRAME_SUB),
+    AP_GROUPINFO_FRAME("_SPEC_GRAV", 8, AP_Baro, _specific_gravity, 1.0, AP_PARAM_FRAME_SUB),
 
 #if BARO_MAX_INSTANCES > 1
-    // @Param: ABS_PRESS2
-    // @DisplayName: Absolute Pressure
+    // @Param: 2_GND_PRESS
+    // @DisplayName: Ground Pressure
     // @Description: calibrated ground pressure in Pascals
     // @Units: Pa
     // @Increment: 1
     // @ReadOnly: True
     // @Volatile: True
     // @User: Advanced
-    AP_GROUPINFO("ABS_PRESS2", 9, AP_Baro, sensors[1].ground_pressure, 0),
+    AP_GROUPINFO_FLAGS("2_GND_PRESS", 9, AP_Baro, sensors[1].ground_pressure, 0, AP_PARAM_FLAG_INTERNAL_USE_ONLY),
 
     // Slot 10 used to be TEMP2
 #endif
 
 #if BARO_MAX_INSTANCES > 2
-    // @Param: ABS_PRESS3
+    // @Param: 3_GND_PRESS
     // @DisplayName: Absolute Pressure
     // @Description: calibrated ground pressure in Pascals
     // @Units: Pa
     // @Increment: 1
     // @ReadOnly: True
     // @Volatile: True
     // @User: Advanced
-    AP_GROUPINFO("ABS_PRESS3", 11, AP_Baro, sensors[2].ground_pressure, 0),
+    AP_GROUPINFO_FLAGS("3_GND_PRESS", 11, AP_Baro, sensors[2].ground_pressure, 0, AP_PARAM_FLAG_INTERNAL_USE_ONLY),
 
     // Slot 12 used to be TEMP3
 #endif
 
-    // @Param: FLTR_RNG
+    // @Param: _FLTR_RNG
     // @DisplayName: Range in which sample is accepted
```

Comment:
An existing problem but this display name is pretty terrible.  It gives no indication to the user what it's affecting without looking at the actual parameter name.